config/initializers/docker.rb

```diff
@@ -9,5 +9,5 @@
     BetterErrors::Middleware.allow_ip!(host_ip)
   end
 
-  Rails.application.config.web_console.whitelisted_ips << host_ip
+  Rails.application.config.web_console.permissions = host_ip
```

Comment:
Why did you choose to replace the shovel with the equals here?

Reply:
Because of the documentation: 

```ruby
Rails.application.configure do
  config.web_console.permissions = '192.168.0.0/16'
end
```

in https://github.com/rails/web-console#configweb_consolepermissions

technically it can receive an array, so we could use `<<` but by default is nil and as it's only used there, my laziness won :D